internal/installation/install.go

```diff
@@ -93,22 +91,24 @@ func Install(p environment.Paths, plugin index.Plugin, opts InstallOpts) error {
 
 func install(op installOperation, opts InstallOpts) error {
 	// Download and extract
-	klog.V(3).Infof("Creating download staging directory %q", op.downloadStagingDir)
-	if err := os.MkdirAll(op.downloadStagingDir, 0755); err != nil {
-		return errors.Wrapf(err, "could not create staging dir %q", op.downloadStagingDir)
+	klog.V(3).Infof("Creating download staging directory")
```

Comment:
I think we also need to delete this dir after we are done right?

Maybe defer some cleanup?

Reply:
@ahmetb there is a defer to clean the dir at the line 100 when we are sure that the directory has been created.

Reply:
sorry for missing :D